src/utils/convertTimeStampToHrMin.js

```diff
@@ -0,0 +1,8 @@
+const convertTimeStampToHrMin = timestamp => {
+  const date = new Date(timestamp);
+  const hour = ('0' + date.getHours()).slice(-2);
+  const mins = ('0' + date.getMinutes()).slice(-2);
+  return `${hour}:${mins}`;
```

Comment:
This timestamp is in UTC. So here we need to convert to local time too. We can do this improvement in a separate task